src/NUnitFramework/framework/Constraints/SubstringConstraint.cs

```diff
@@ -39,16 +44,38 @@ public SubstringConstraint(string expected) : base(expected)
         }
 
         /// <summary>
+        /// Modify the constraint to ignore case in matching.
+        /// This will call Using(StringComparison.CurrentCultureIgnoreCase) 
+        /// </summary>
+        public override StringConstraint IgnoreCase
+        {
+            get { caseInsensitive = true; Using(StringComparison.CurrentCultureIgnoreCase); return this; }
```

Comment:
Since this class doesn't "own" the `caseInsensitive` field, perhaps we should be calling `var _ = base.IgnoreCase` instead of coupling to `caseInsensitive`. Probably not important either way.